examples/generate_phpdoc.py

```diff
@@ -31,7 +31,9 @@
     GValue.array_int_type: 'integer[]|integer',
     GValue.array_double_type: 'float[]|float',
     GValue.array_image_type: 'Image[]|Image',
-    GValue.blob_type: 'string'
+    GValue.blob_type: 'string',
+    GValue.source_type: 'VipsSource',
```

Comment:
We don't use the `Vips` prefix on other names (we have `Vips\Image`, not `Vips\VipsImage`), so I think these should just be `Source` and `Target`.

Reply:
... so for example, `VipsImage` is the name of the C class in `FFI.php`, but `Vips\Image` is the name of the php class we wrap it with.